src/voice-broadcast/stores/VoiceBroadcastRecordingStore.ts

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { MatrixClient, MatrixEvent, RelationType } from "matrix-js-sdk/src/matrix";
+import { TypedEventEmitter } from "matrix-js-sdk/src/models/typed-event-emitter";
+
+import { VoiceBroadcastInfoEventType, VoiceBroadcastInfoState } from "..";
+
+export enum VoiceBroadcastRecordingStoreEvent {
+    StateChanged = "liveness_changed",
+}
+
+interface EventMap {
+    [VoiceBroadcastRecordingStoreEvent.StateChanged]: (state: VoiceBroadcastInfoState) => void;
+}
+
+/**
+ * This store handles one Voice Broadcast recording.
+ */
+export class VoiceBroadcastRecordingStore extends TypedEventEmitter<VoiceBroadcastRecordingStoreEvent, EventMap> {
```

Comment:
why `Store`? The purpose of this class would be more obvious to me if it were just `VoiceBroadcastRecording`

Reply:
One could say it stores the info about a Voice Broadcast. On the other hand it could be a „model“…

Moving it to `models` :heavy_check_mark: